test/models/video_test.rb

```diff
@@ -1,7 +1,60 @@
 require "test_helper"
 
 describe Video do
-  # it "does a thing" do
-  #   value(1+1).must_equal 2
-  # end
+  describe "validations" do
+    before do
+      @video = videos(:grand_budapest)
+    end
+
+  it "is invalid without a title" do
+    @video.title = nil
+
+    result = @video.valid?
+
+    expect(result).must_equal false
+    expect(@video.errors.messages).must_include :title
+    expect(@video.errors.messages[:title]).must_equal ["can't be blank"]
+  end
+
+  it "is invalid without an overview" do
+    @video.overview = nil
+
+    result = @video.valid?
+
+    expect(result).must_equal false
+    expect(@video.errors.messages).must_include :overview
+    expect(@video.errors.messages[:overview]).must_equal ["can't be blank"]
+  end
+
+  it "is invalid without a release date" do
+    @video.release_date = nil
+
+    result = @video.valid?
+
+    expect(result).must_equal false
+    expect(@video.errors.messages).must_include :release_date
+    expect(@video.errors.messages[:release_date]).must_equal ["can't be blank"]
+  end
+
+  it "is invalid without a total inventory" do
+    @video.total_inventory = nil
+
+    result = @video.valid?
+
+    expect(result).must_equal false
+    expect(@video.errors.messages).must_include :total_inventory
+    expect(@video.errors.messages[:total_inventory]).must_equal ["can't be blank"]
+  end
+
+  it "is invalid without an available inventory" do
```

Comment:
Do we wanna test the greater_than_or_equal_to and only_integer validations too? I tested these for customer.videos_checked_out_count but was wondering if it was a little overkill... 😅

Feel free to make this call! I think this test does the job as is.